Improve installation documentation (#135)
sulu/SuluCommunityBundle: Resources/doc/1-installation.md

    prefix: /admin/api
```

## Configure security

> The security needs now be configure in the `security_website.yaml`:

Reviewer: ```suggestion
Enable and configure the security for the website in the `security_website.yaml`:
```

    providers:
        sulu:
            id: sulu_security.user_provider

    access_control:
        # needed when firewall on ^/ is not anonymous
        # - { path: '/login', roles: IS_AUTHENTICATED_ANONYMOUSLY }
        # - { path: '/registration', roles: IS_AUTHENTICATED_ANONYMOUSLY }
        # - { path: '/password-reset', roles: IS_AUTHENTICATED_ANONYMOUSLY }
        # - { path: '/password-forget', roles: IS_AUTHENTICATED_ANONYMOUSLY }
        # - { path: '/_fragment', roles: IS_AUTHENTICATED_ANONYMOUSLY }
        - { path: '/profile', roles: ROLE_USER }
        - { path: '/completion', roles: ROLE_USER }

    firewalls:
        <webspace_key>: # Replace <webspace_key> with the key of your webspace
            pattern: ^/
            anonymous: lazy
            form_login:
                login_path: sulu_community.login
                check_path: sulu_community.login
            logout:
                path: sulu_community.logout
                target: /
            remember_me:
                secret:   "%kernel.secret%"
                lifetime: 604800 # 1 week in seconds
                path:     /

sulu_security:
    checker:
        enabled: true
```

> For functional tests you need to activate the security in the website test configuration:

Reviewer: ```suggestion
When implementing functional tests, website security needs to enabled in the test environment:
```

```yaml
# config/packages/test/security_website.yml

security:
    providers:
        sulu:
            id: test_user_provider

    firewalls:
        <webspace_key>: # Replace <webspace_key> with the key of your webspace
            http_basic: ~

sulu_test:
    enable_test_user_provider: true
```

## Clear cache

> That the new config is loaded the caches need to be cleared:

Reviewer: ```suggestion
In order to use the new configuration, the caches need to be cleared:
```
